src/errors.js

```diff
@@ -0,0 +1,6 @@
+const got = require('got');
+
+module.exports = {
+    TIMEOUT_ERR: got.TimeoutError,
```

Comment:
Why `TIMEOUT_ERR` and not `TIMEOUT_ERROR` ? I think we can spend those 2 more bytes and keep the same name

Reply:
Also, why all caps, it's just a regular class, isn't it?

Reply:
Yeah, I agree. I got carried away by the constants.